tests/integration/examples/test_iteration_with_cluster.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from adcm_aio_client import ADCMSession, Credentials
+from tests.integration.setup_environment import ADCMContainer
+
+pytestmark = [pytest.mark.asyncio]
+
+
+async def test_iteration_with_cluster(adcm_for_examples: ADCMContainer) -> None:
+    """
+    Interaction with clusters: creating, deleting, getting a list of clusters using filtering,
+    configuring cluster configuration, launching actions on the cluster and updating the cluster.
+    """
+    credentials = Credentials(username="admin", password="admin")  # noqa: S106
+    url = adcm_for_examples.url
```

Comment:
tip: if you move `adcm_for_examples` into `conftest.py` into `examples`, you'll be able to override `adcm` fixture and use more obvious names for users

Reply:
Renamed it